packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1235,13 +1235,6 @@ class Transform extends SingleChildRenderObjectWidget {
   ///
   /// This is equivalent to setting an origin based on the size of the box.
   /// If it is specified at the same time as the [origin], both are applied.
-  ///
-  /// An [AlignmentDirectional.centerStart] value is the same as an [Alignment]
```

Comment:
You're removing this because the reader can find this out by clicking through to `AlignmentGeometry` (and then transitively through to [AlignmentDirectional])?

Reply:
I actually didn't mean to remove this. Putting an updated version back in.